qtism/data/storage/xml/marshalling/OutcomeDeclarationMarshaller.php

```diff
@@ -115,7 +119,12 @@ protected function unmarshall(DOMElement $element) {
 			$object->setBaseType($baseComponent->getBaseType());
 			$object->setCardinality($baseComponent->getCardinality());
 			$object->setDefaultValue($baseComponent->getDefaultValue());
-			
+
+			// Set external scored attribute
+			if ($element->hasAttribute('externalScored')) {
```

Comment:
do u have a constant for that :) ?

Reply:
Leave the hard-coded string now.
There's a lot of cleaning I'm doing now to merge legacy branch back into develop, and I'd like to keep the changes at the minimum. We can optimize the code once it's merged.